README.md

```diff
@@ -16,7 +16,7 @@ the square brackets (for multiple-choice questions).
 What are constructor functions and the `new` keyword? What is a prototype? Describe an example of when we would use a constructor function versus a prototype.
 
 Your Answer:
-```text
+```The constructor function creates an object prototype. When you say NEW, you're calling the constructor function and you create a new object... I think .
```

Comment:
It doesn't create an object prototype; it creates an object *from* the prototype. Aside from that, you got it.